src/V3EmitCImp.cpp

```diff
@@ -909,7 +908,7 @@ class EmitCTrace final : EmitCFunc {
     ~EmitCTrace() override = default;
 
 public:
-    static void main(AstNodeModule* modp, bool slow, std::deque<AstCFile*>& cfilesr) {
+    static void main(AstNodeModule* modp, bool slow, std::deque<AstCFile*>& cfilesr) VL_MT_STABLE {
         EmitCTrace{modp, slow, cfilesr};
```

Comment:
None of the functions that will be called by EmitCTrace are marked stable, so this shouldn't be stable (and we need better checks to flag this).

If you want, separate out where you added VL_MT_STABLE's then this can be merged.